sphinx_antsibull_ext/__init__.py

```diff
@@ -28,8 +28,7 @@ def setup(app):
     for lexer in [
```

Comment:
The outer loop with a single item is unnecessary: it just adds nesting for no reason

Reply:
It's to make it easier to add new parsers, without having to do error-prone copy'n'paste.

Reply:
I doubt there will be new parsers anytime soon. atm it's just a premature optimization.